### ISSUE TITLE
Big battery icon - budgie-desktop

### ISSUE THREAD

**Issue by fossfreedom** [numixproject/numix-icon-theme#1048]
split from #1046 

![](http://pix.toile-libre.org/upload/original/1459630350.png)

Top right - the icon sizing is a little off.  Not sure if this is a budgie-desktop issue or numix.

Battery Icon looks ok under Moka/Faba icon theme.

Confirmed that a discharging and charging icon is similarly larger than the rest.

Looking at this bit of code line 77 onwards there are a variety of icon names used - https://github.com/solus-project/budgie-desktop/blob/master/panel/applets/status/PowerIndicator.vala

As far as I can see line 100 onwards adds icons without padding

**Comment by palob** [comment]
This is what it looks like in GNOME:
![battery](https://cloud.githubusercontent.com/assets/7050624/14232827/e5affbae-f9b6-11e5-98c9-563b2ab03dd5.png)

Due to its shape the icon looks comparably big indeed.

The one in #1047 looks even bigger though but given it doesn't look blurry I don't think it's scaled up.

**Comment by dirtydancing** [comment]
Confirmed that the battery icon is not scaled up. I checked in comparison with Arch/GNOME Shell. The battery icon size is exactly the same, although the icon looks slimmer with GNOME Shell because the spacing inbetween the icons is bigger.

It seems that similar to GNOME, Budgie is using `-symbolic`- named battery icons. This would mean that this is not a Numix issue, but rather a Budgie issue of the spacing inbetween the icons in the top bar.

Screenshot with GNOME:
![battery_icon_gnome-shell](https://cloud.githubusercontent.com/assets/7164227/14233052/aa1abb18-f9bd-11e5-8af2-c2b2100c2218.png)

Size comparison Budgie (screenshot from above) - GNOME (identical icon size):
![battery_icon_comparison](https://cloud.githubusercontent.com/assets/7164227/14233060/ce3859c4-f9bd-11e5-8ad9-c678bd850c33.png)
